app/javascript/packs/application.js

```diff
@@ -33,4 +34,5 @@ document.addEventListener('turbolinks:load', () => {
   // Call your functions here, e.g:
   initSelect2();
   initAutoFadeAlert();
+  if (document.getElementById('address-for-autocomplete')) initAutocomplete();
```

Comment:
@Nicode80, I don't think you need a` if` here since you already have a guard close in your init_autocomplete.js. If we really need it, can you explain why ? Otherwise, please it.

Reply:
done